### DISCUSSION TITLE
How to view soft deleted user?

### DISCUSSION THREAD

**Discussion by faizananwerali** [filamentphp/filament#8285] [Help]
How to view soft deleted users?

If you have a User Model with soft deletes and a view page on UserResource, if the user is soft deleted then the edit, view or any user page won't be available, it will give a 404 error. how do I view the page even with soft deleted users, to allow the admin to restore them?

**Comment by leandrocfe** [comment]
[creating-a-resource-with-soft-delete](https://filamentphp.com/docs/3.x/panels/resources/deleting-records#creating-a-resource-with-soft-delete) ?

**Comment by VasasA** [comment]
@faizananwerali You can use the `getEloquentQuery()` in the UserResource.  In the [official documentation](https://filamentphp.com/docs/3.x/panels/resources/deleting-records#adding-soft-deletes-to-an-existing-resource):
```php
public static function getEloquentQuery(): Builder
{
    return parent::getEloquentQuery()
        ->withoutGlobalScopes([
            SoftDeletingScope::class,
        ]);
}
```
